package.json

```diff
@@ -68,6 +69,7 @@
     "http-proxy-agent": "^7.0.2",
     "https-proxy-agent": "^7.0.6",
     "lodash": "4.17.21",
+    "lru-cache": "^11.0.2",
```

Comment:
Now that we moved it to a package, we dont need this anymore right?

Reply:
Yep i'll remove it